compiler/damlc/tests/src/DA/Test/Packaging.hs

```diff
@@ -295,6 +295,17 @@ tests damlc = testGroup "Packaging"
             , "dependencies: [daml-prim, daml-stdlib]"
             ]
         buildProject projDir
+
+    , testCase "Empty package" $ withTempDir $ \projDir -> do
+        writeFileUTF8 (projDir </> "daml.yaml") $ unlines
+            [ "sdk-version: " <> sdkVersion
+            , "name: proj"
+            , "version: 0.0.1"
+            , "source: src"
+            , "dependencies: [daml-prim, daml-stdlib]"
```

Comment:
What exactly in an empty package depends on these libraries? ;)

Reply:
Even if something depends on them, the entries are meaningless since we filter those two libs out :slightly-smiling-face: